From 9d7931c8c2875c0ffc03927ffb9aad9648df88cb Mon Sep 17 00:00:00 2001
From: Owen D'Aprile <git@owen.sh>
Date: Sun, 25 Sep 2022 15:12:23 -0400
Subject: [PATCH] linux: don't use hardcoded path for Exec and Icon

Desktop files respect $PATH and $XDG_DATA_DIRS, so the full path does
not need to be specified to either.

This fixes the Flatpak build by allowing Flatpak to rewrite these to
match the app ID.
---
 desktop/samplebrain.desktop | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/desktop/samplebrain.desktop b/desktop/samplebrain.desktop
index 76b77e6..20d400b 100644
--- a/desktop/samplebrain.desktop
+++ b/desktop/samplebrain.desktop
@@ -3,8 +3,8 @@ Encoding=UTF-8
 Type=Application
 Name=Samplebrain
 Comment=A sample masher designed by Aphex Twin
-Exec=/usr/bin/samplebrain
-Icon=/usr/share/icons/hicolor/scalable/apps/samplebrain.svg
+Exec=samplebrain
+Icon=samplebrain
 Terminal=false
 Categories=GNOME;Application;
 StartupNotify=true
\ No newline at end of file
-- 
GitLab

